custom_search_commands/README.md

```diff
@@ -0,0 +1,36 @@
+# Overview of custom search commands
+
+Custom search commands are user-defined Splunk Search Processing Language (SPL) commands implemented in Python scripts that extend SPL to serve your specific needs. Custom search commands let you perform additional data analysis in Splunk Cloud Platform or Splunk Enterprise. 
+
+## Use cases for custom search commands
+
+Here are some use cases for custom search commands.
+
+* You want to process data in a way that Splunk software hasn't handled yet.
+* You want to import data from an external source that is not indexed by Splunk software into your search pipeline.
+* You want to export search results to an external system, such as JIRA.
+
+## Workflow for creating a custom search command
+
+Follow this workflow to create a custom search command.
+
+1. Install the Splunk SDK for Python in your app.
```

Comment:
I might reword "Install the Splunk SDK for Python in your app" as this seems a little odd to me, though not exactly incorrect. More like "Install the Splunk SDK for Python for use with your app" because I think the typical use case is to install the SDK onto your Enterprise system using pip once for any/all app development, then as needed, one imports libs/resources into their app. The SDK itself doesn't wholly go with the app, if that makes sense.

Reply:
That's really interesting, because we actually instruct people to install the SDK directly in their app's /lib directory. Is this unnecessary if they already have the SDK installed to use with other apps? I do agree that for this, "for use with your app" is more clear.